### PR TITLE
Added: Steal the fuel truck

### PR DESCRIPTION
**Pull request by killerswin2**
## What type of PR is this.
1. [ ] Bug
2. [x] Change
3. [x] Enhancement

### What have you changed and why?
Information:
    This was supposed to be added during fuel reworks

### Please verify the following and ensure all checks are completed.

1. [x] Have you loaded the mission in LAN host?
2. [ ] Have you loaded the mission on a dedicated server?

### Is further testing or are further changes required?
1. [x] No
2. [ ] Yes (Please provide further detail below.)

### How can the changes be tested?
Steps: 

********************************************************
Notes:

### CODE REVIEW

File: A3A/addons/core/Stringtable.xml [hunk 1]
```diff
@@ -5055,6 +5055,9 @@
         <Czech>Ukradni nebo Znič muniční náklaďák</Czech>
         <Chinesesimp>盗取或者摧毁敌方弹药车</Chinesesimp>
       </Key>
+      <Key ID="STR_A3A_fn_mission_log_ammo_stolen_text">
+        <Original>Ammotruck Stolen in an %1</Original>
```
Comment (rautamiekka) [inline A3A/addons/core/Stringtable.xml:5059]:
```suggestion
        <Original>Ammo truck stolen in an %1</Original>
```

File: A3A/addons/core/Stringtable.xml [hunk 2]
```diff
@@ -5112,6 +5115,15 @@
...
         <Chinesesimp>银行劫案</Chinesesimp>
       </Key>
+      <Key ID="STR_A3A_fn_mission_log_fuel_text">
+        <Original>We've spotted a Fuel truck in %1. Go there and destroy or steal it before %2.</Original>
+      </Key>
+      <Key ID="STR_A3A_fn_mission_log_fuel_title">
+        <Original>Steal or Destroy Fuel truck</Original>
+      </Key>
+      <Key ID="STR_A3A_fn_mission_log_fuel_stolen_text">
+        <Original>Fuel truck Stolen in an %1</Original>
```
Comment (rautamiekka) [inline A3A/addons/core/Stringtable.xml:5125]:
```suggestion
        <Original>Fuel truck stolen in an %1</Original>
```

**Review by rautamiekka — CHANGES_REQUESTED** [review]

**Review by jaj22 — COMMENTED** [review]
Looks fine generally, although as this change halves the number of ammo truck missions, we might need to balance by adding more ways to acquire gear.

**Comment by killerswin2** [comment]
the fuel truck still has the ammo in it.

**Comment by ante185** [comment]
Don't add ammo/gear loot to the fuel truck.
Make the fuel truck mission less probable than the ammo truck mission, ie select random weighted or if random > x or something.
additional suggestion is to make the fuel truck mission more likely when there's none in the civilian template / less likely if there's fuel trucks in the civilian template

**Comment by killerswin2** [comment]
> Don't add ammo/gear loot to the fuel truck. Make the fuel truck mission less probable than the ammo truck mission, ie select random weighted or if random > x or something. additional suggestion is to make the fuel truck mission more likely when there's none in the civilian template / less likely if there's fuel trucks in the civilian template

no, that was an unintended feature

**Pull request closed without merging**